### PR TITLE
Fixed Inconsistent Label spacing on PDP

### LINKED ISSUE #3042: Inconsistent label spacing on pdp
### Expected behavior
Labels should have consistent spacing 

### Actual Behavior
Labels have inconsistent spacing. 

spacing-bug

### Steps to Reproduce the Behavior
Create a new product with both backorder and archived labels. 

### Versions
Node: 4.5.0
NPM: 
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.13.1
Reaction: 1.5.0
Reaction branch: marketplace

### PR DESCRIPTION
**Pull request by aanchirinah**
Resolves #3042 

 I have added a custom style class in `products.less` to fix the label badges anomaly.

#### How to test
- Login in as admin
- Add a new product 
- Enable Inventory tracking
- Set `Warn at`  value to be more than the `Quantity` value to trigger getting the `Backorder` label badge
- Archive the variant to also trigger the `Archived` label badge.
- Observe that the labels are currently displayed with the desired spaces.

### CODE REVIEW

File: imports/plugins/included/product-detail-simple/client/components/childVariant.js [hunk 1]
```diff
@@ -50,14 +50,14 @@ class ChildVariant extends Component {
     if (inventoryManagement && this.props.variant.inventoryQuantity <= 0) {
       if (inventoryPolicy) {
         return (
-          <span className="variant-qty-sold-out badge badge-danger">
+          <span className="variant-qty-sold-out badge badge-danger" style={{ marginLeft: "5px" }}>
```
Comment (mikemurray) [inline imports/plugins/included/product-detail-simple/client/components/childVariant.js]: No inline styles. This needs to go into a LESS file somewhere. Probably one of these files in this folder `/imports/plugins/included/default-theme/client/styles/products`

File: imports/plugins/included/product-detail-simple/client/components/childVariant.js [hunk 2]
```diff
@@ -50,14 +50,14 @@ class ChildVariant extends Component {
...
             <Components.Translation defaultValue="Sold Out!" i18nKey="productDetail.soldOut" />
           </span>
         );
       }
 
       return (
-        <span className="variant-qty-sold-out badge badge-info">
+        <span className="variant-qty-sold-out badge badge-info" style={{ marginLeft: "5px" }}>
```
Comment (mikemurray) [inline imports/plugins/included/product-detail-simple/client/components/childVariant.js]: No inline styles.

File: imports/plugins/included/product-detail-simple/client/components/variant.js [hunk 3]
```diff
@@ -9,6 +9,20 @@ import { ReactionProduct } from "/lib/api";
 
 import { ProductVariant } from "/lib/collections/schemas";
 
+const customStyle = {
```
Comment (mikemurray) [inline imports/plugins/included/product-detail-simple/client/components/variant.js]: No inline styles, add this is a CSS file somewhere, most likely one of the files here `/imports/plugins/included/default-theme/client/styles/products`

**Comment by aanchirinah** [comment]
@mikemurray , I have fixed the requested changes.

**Comment by aanchirinah** [comment]
@mikemurray , thanks immensely for your input. Inclusion of `align-items: center` fixed it and I just maintained the margin in the custom style.

**Comment by aanchirinah** [comment]
@mikemurray , I have fixed the requested changes using LESS mixins like you recommended.

File: imports/plugins/included/default-theme/client/styles/products/products.less [hunk 4]
```diff
@@ -96,3 +96,16 @@
 .product-load-more {
   padding: 0 60px;
 }
+
+.custom-badge-variant {
```
[the remark this answers is not in the document]
Comment (aanchirinah) [reply imports/plugins/included/default-theme/client/styles/products/products.less]: After our discussion I have removed all `custom` prefixes in my style naming

**Comment by zenweasel** [comment]
@rymorgan When you get a chance can you make sure this is what we wanted?

**Comment by aaronjudd** [comment]
@mikemurray can you revisit your review here and update?
@rymorgan any input?

**Comment by rymorgan** [comment]
The size of the label looks fixed. I'm seeing some other funky issues that I don't see on 1.5.3 including this archiving issue below. I also don't see the backorder label show up on a variant options on this branch. Not sure what's happening here. 

pdp-1

**Comment by aanchirinah** [comment]
@rymorgan , I just pulled the recent changes from master and merged to this branch, It works as it should. Can you test it out again.

**Comment by iakhator** [comment]
I tested this and it is working properly.

**Comment by rymorgan** [comment]
Looks good to me.

**Comment by spencern** [comment]
@aanchirinah I'd like to get this merged into 1.5.6 tomorrow, but it's failing tests. Can you check into that? I'm re-running now, but if there's a real problem, we'll need to get that addressed before merging.

**Comment by aanchirinah** [comment]
@spencern , I just tested it out again and there are no failing tests , could you please try again

**Pull request merged**